src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs

```diff
@@ -65,6 +65,7 @@ private async void FileTagItem_ItemClick(object sender, ItemClickEventArgs e)
 		private void Item_RightTapped(object sender, RightTappedRoutedEventArgs e)
 		{
 			var itemContextMenuFlyout = new CommandBarFlyout { Placement = FlyoutPlacementMode.Full };
+			itemContextMenuFlyout.Opening += (sender, e) => App.LastOpenedFlyout = sender as CommandBarFlyout;
```

Comment:
Should we unsubscribe when it closes?

Reply:
I think there is no need to do so because a new instance of the flyout is created each time and is not reused.

Reply:
I have verified that the flyout works after showing flyouts several times on the home screen and also that the application exits without issue. Should I still check something?

Reply:
Memory leaks from not subscribing would occur if the publisher continues to remain. But since the publisher is disposed every time, memory leaks should not occur. If a memory leak were to occur, it would happen without subscribing to the event.